docs/dashboards/chart-builder.md

```diff
@@ -24,16 +24,37 @@ style:
  
 Each series should be configured in a series configuration panel. The panel where you change the settings of each series is split into a couple of sections.
 
-## Metrics
+## Data source
 
-In this section you pick one or more metrics which are used to build a data series. If you pick more
-than one metric you have to apply transformations to let Chart Builder know what's
+With Chart Builder you can create charts and other visual component for all App types
```

Comment:
With Chart Builder you can create charts and other visual component for all App types
=>
With Chart Builder you can create charts and other visual components for all App types